src/Config.php

```diff
@@ -57,13 +57,20 @@ class Config implements ConfigInterface
      *
      * @var array<string, array<string, mixed>|bool>
      */
-    private array $rules = ['@PSR12' => true];
+    private array $rules;
 
     private bool $usingCache = true;
 
     public function __construct(string $name = 'default')
     {
-        $this->name = $name;
+        // @TODO 4.0 cleanup
+        if (getenv('PHP_CS_FIXER_FUTURE_MODE')) {
```

Comment:
```suggestion
        if (filter_var(getenv('PHP_CS_FIXER_FUTURE_MODE'), FILTER_VALIDATE_BOOL)) {
```

In order to handle several `true`-like values.

Reply:
good idea, but we need in more places: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7651